backend/package.json

```diff
@@ -221,7 +221,7 @@
     "lodash.isequal": "^4.5.0",
     "mongodb": "^6.8.1",
     "ms": "^2.1.3",
-    "mysql2": "^3.9.8",
+    "mysql2": "^3.15.2",
```

Comment:
were you able to test dynamic secrets and secret rotation after this version bump?

Reply:
okay, just revert the mysql2 version